hack/scripts/replace-module-path.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+
+prow="$PWD/prow"
+cd "$prow" || exit
+
+# replace go module name
+find . -type f -name "*.go" -exec sed -i.bak 's/k8s.io\/test-infra/sigs.k8s.io\/prow/g' {} \;
+find . -type f -name "*.go.bak" -delete # Remove backup files
+
+# restore imports using packages from the test-infra repo
```

Comment:
Comments should try to explain more than what is happening. For example, *why* are you undoing the sed work from above to use the old import paths? The answer to that question would be very helpful for readers/reviewers. Thanks.